io.github.Figma_Linux.figma_linux.desktop

```diff
@@ -0,0 +1,11 @@
+
+[Desktop Entry]
```

Comment:
Ideally you should maintain it in upstream repo.

Reply:
Yes, sure